### DISCUSSION TITLE
The side by side compare component not found

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [dubinc/dub#1198] [Q&A] [answered]
I tried to look into the repo for the compare component but couldn't find it, isn't it open-source?

**Comment by Subr1ata** [comment]
> I tried to look into the repo for the compare component but couldn't find it, isn't it open-source?

I am new to this project, can you help me to understand this project ?

**Comment by steven-tey** [comment, marked as answer]
@Ali-Hussein-dev if you're referring to the compare component on dub.co/compare/bitly, that's not open-source. Everything under app.dub.co, along with all our short link redirect proxies are open-source in this repo. Hope that helps!

**Reply by Ali-Hussein-dev** [reply]
Thanks for answering. Have you used a library for that? if yes can you share its name?
